samples/application-mobility/csm_application_mobility_with_driver.yaml

```diff
@@ -471,16 +466,16 @@ spec:
             name: cloud-creds
             #Specify the object store access credentials to be stored in a secret with key "cloud".
             secretContents: 
-              aws_access_key_id:   #Provide the access key id here
-              aws_secret_access_key: #provide the access key here 
+              aws_access_key_id:  #Provide the access key id here
+              aws_secret_access_key:  #provide the access key here 
               
 
     # Init containers to be added to the Velero and Node-agent deployment's spec.
     - initContainer:
         #initContainer image for the dell velero plugin
       - name: dell-custom-velero-plugin
-        image:
+        image: 
```

Comment:
Why did we remove plugin images here ?

Reply:
it was the same before

Reply:
yeah, there is no public plugin image to use
We will fill it in when we have it posted on DockerHub